ansible/playbooks/roles/repository/files/download-requirements/src/downloader.py

```diff
@@ -54,18 +54,22 @@ def __is_checksum_valid(self, requirement: str, requirement_file: Path) -> bool:
 
         return True
 
-    def download(self, requirement: str, requirement_file: Path, sub_key: str = None):
+    def download(self, requirement: str, requirement_file: Path, sub_key: str = None, additional_args: dict = None):
```

Comment:
dict -> Dict
You must use `Dict` from the `typing` module because this code is ran on a repository machine which has python in version 3.7.
Native typing is supported with python3.10+